Gemfile.lock

```diff
@@ -200,39 +221,45 @@ GEM
     puma (6.4.0)
       nio4r (~> 2.0)
     racc (1.7.1)
-    rack (2.2.8)
-    rack-protection (3.1.0)
-      rack (~> 2.2, >= 2.2.4)
+    rack (3.0.8)
+    rack-protection (3.0.6)
```

Comment:
rack-protection requires rack 3 support.
- https://github.com/sinatra/sinatra/pull/1857

3.0.6 officially unsupports rack 3, but there are no change related between rack 2 and 3.
Since it works fine for as expected by starseeker, we allow this downgrade.